kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-common/src/main/java/org/kie/server/remote/rest/common/resource/KieServerRestImpl.java

```diff
@@ -126,6 +138,24 @@ public Response listContainers(@Context HttpHeaders headers,
         return createCorrectVariant(server.listContainers(containerFilter), headers);
     }
 
+    @GET
+    @Path("prometheus")
```

Comment:
maybe better call it metrics instead of prometheus ... more generic and provides bit more info that product/project name that might change.

Reply:
Done